mne_bids/path.py

```diff
@@ -52,37 +54,48 @@ class BIDSPath(object):
         The recording name for this item. Corresponds to "rec".
     space : str | None
         The coordinate space for an anatomical file. Corresponds to "space".
-    prefix : str | None
-        The prefix for the filename to be created. E.g., a path to the folder
-        in which you wish to create a file with this name.
-    suffix : str | None
-        The suffix for the filename to be created. E.g., 'audio.wav'.
+    kind : str | None
+        The recording kind. Will be appended to the filename
+        before the extension. E.g., 'ieeg'.
+    extension : str | None
```

Comment:
call it `ext` following convention in `os` module?

Reply:
I suggested to call it `extension` for consistency with the other kwargs we're using here – none of them is abbreviated anymore.

Reply:
I'm in favor of `extension` for consistency. We could also add `ext` as an alias for `extension`?

Reply:
> I'm in favor of `extension` for consistency. We could also add `ext` as an alias for `extension`?

Please no aliases / two parameter names for the same thing. 
+1 for using `extension` for consistency